.pylintrc

```diff
@@ -11,6 +11,8 @@ disable=too-many-arguments,
   fixme,
   import-error,
   unspecified-encoding,
+  too-few-public-methods,
+  import-outside-toplevel,
 
 [FORMAT]
 max-line-length=100
```

Comment:
I would suggest increasing the max-line-lenght to 200 for fewer restrictions.

Reply:
200 seems too long to me. The standard for PyCharm is 120, is that a suitable compromise?

Reply:
I also think, that 120 should be enough.